hydra-node/src/Hydra/Chain/ZeroMQ.hs

```diff
@@ -155,17 +156,20 @@ catchUpTransactions catchUpAddress handler tracer = runZMQ $ do
       forM_ txs handler
     Nothing -> error $ "cannot decode catch-up transactions  " <> show message
 
-createMockChainClient :: Tx tx => EventQueue IO (Event tx) -> Tracer IO (MockChainLog tx) -> IO (Chain tx IO)
-createMockChainClient EventQueue{putEvent} tracer = do
+createMockChainClient ::
+  Tx tx =>
+  MockChainPorts ->
+  EventQueue IO (Event tx) ->
+  Tracer IO (MockChainLog tx) ->
+  IO (Chain tx IO)
+createMockChainClient (MockChainPorts (syncPort, catchUpPort, postPort)) EventQueue{putEvent} tracer = do
   -- TODO: Do a proper cleanup of threads and what not
   -- BUG(SN): This should wait until we are connected to the chain, otherwise we
   -- might think that the 'OnChain' is ready, but it in fact would not see any
   -- txs from the chain. For now, we assume it takes 1 sec to connect.
```

Comment:
I just realize that this comment got out of date :( seems to have been changed to `0.1` .. which I think its quite low to be certain that we see all txs?

Reply:
The comment is slightly out of date yes, but the general idea still relevant I guess. I will modify it.